### PR TITLE
Explanation for contents of harmonic analysis output files

### PR DESCRIPTION
**Pull request by fsoubelet**
This is the small block explaning the contents of the files output by `harpy` that was missing in the `omc3` Worfklow page (now renamed Analysis).

If someone knows what the remaining columns mean (`FREQ1_3` etc) please contribute!

### CODE REVIEW

File: docs/packages/omc3/analysis.md [hunk 1]
```diff
@@ -770,7 +770,37 @@ trackone.sdds.bad_bpms_y	trackone.sdds.liny
...
+    for bpm in ampsx.columns:
+        if bpm in freqsx.columns:  # safety check but no reason it wouldn't be there
+            plt.plot(freqsx[bpm], ampsx[bpm], ".-") 
+            # plt.stem(freqsx[bpm], ampsx[bpm])  # would be more accurate but might stress your system
+
+    plt.xlim(0, 0.5)
+    plt.xlabel(r"$Q_x$")
+    plt.ylabel("Amplitude")
+    plt.title("Horizontal Spectrum")
+    ```
```
Comment (JoschD) [inline docs/packages/omc3/analysis.md:794]: Maybe use the plot_spectrum instead?

File: docs/packages/omc3/analysis.md [hunk 2]
```diff
@@ -770,7 +770,37 @@ trackone.sdds.bad_bpms_y	trackone.sdds.liny
     To use these, refer to the [Plot Spectrum][plot_spectrum] API documentation.
 
 ??? question "Column Nomenclature"
-    TODO: add a reference with the meaning of each column in the important files (the `.lin*` ones)
+    The `*.amps[xy]`, `*.freqs[xy]` and `*.lin[xy]` files in the harmonic analysis output are **TFS** files.
+
+    The `*.freqs[xy]` files contain for each BPM in column format the frequencies of the resonance lines detected in the spectrum, for respectively the horizontal (`.freqsx`) and vertical (`.freqsy`) planes while the `*.amps[xy]` files contain the amplitudes of said resonance lines.
+    This means in the column of a given BPM, the `nth` value in the `.amps[xy]` file corresponds to the amplitude of the resonance line located at the frequency given by the `nth` value in the column of the same name in the `.freqs[xy]` file.
```
Comment (JoschD) [inline docs/packages/omc3/analysis.md:776]: in the `nth` row maybe?

File: docs/packages/omc3/analysis.md [hunk 3]
```diff
@@ -770,7 +770,37 @@ trackone.sdds.bad_bpms_y	trackone.sdds.liny
...
+    plt.xlabel(r"$Q_x$")
+    plt.ylabel("Amplitude")
+    plt.title("Horizontal Spectrum")
+    ```
+
+    The `*.lin[xy]` files contain various data (in columns) computed for each BPM (in rows) as some summary information.
+    Some column names are explicit: `BPM_RES` contains the determined BPM resolution, `CO` the closed orbit value at said BPM and `PK2PK` is the peak-to-peak value of oscillations registered by the given BPM.
+    `TUNE[XY]`, `AMP[XY]` and `MU[XY]` are the detected tune, the amplitude of the tune line and the phase of the tune line for said BPM, respectively.
+    
+    Some columns are named less explicitely but important for later optics calculations.
```
Comment (JoschD) [inline docs/packages/omc3/analysis.md:800]: "The other columns are describe the phase (`PHASE`), amplitude (`AMP`) and frequency (`FREQ`) of the line identified by the number in the column name. Underscores represent a minus sign here."

File: docs/packages/omc3/analysis.md [hunk 4]
```diff
@@ -770,7 +770,37 @@ trackone.sdds.bad_bpms_y	trackone.sdds.liny
...
+    In the `.linx` file `FREQ01`, `AMP01` and `PHASE01` are respectively the frequency of the main vertical tune line in the horizontal spectrum, the ratio of this line to the main horizontal line and the phase of this line.
```
Comment (JoschD) [inline docs/packages/omc3/analysis.md:801]: "Therefore, in the `.linx` file, ...."

File: docs/packages/omc3/analysis.md (hunk 2 above)
Comment (mihofer) [reply docs/packages/omc3/analysis.md:776]: or with an example, e.g loading them as tfs_df and say that freq_df[10, "BPM123"] has the amplitude amp_df[10, "BPM123"]

**Comment by mihofer** [comment]
> If someone knows what the remaining columns mean (`FREQ1_3` etc) please contribute!

Based on the tune for the BPM, the frequency (& amplitude) of higher order lines is calculated (e.g sextupole lines) using m Qx + nQy . 
So for your example, this then translates to m=1, n=-3 (underscores count as minus). For this case abs(m)+abs(n)=4, which should be a decapolar line. See e.g. https://cds.cern.ch/record/333077/files/p93.pdf for more information

File: docs/packages/omc3/analysis.md (hunk 1 above)
Comment (fsoubelet) [reply docs/packages/omc3/analysis.md:794]: I want this to be a simple example that shows the plotting and illustrates the point of the sentences above. I would rather leave this as is and mention below that we have a dedicated spectrum plotter to take care of all of this for us.

File: docs/packages/omc3/analysis.md (hunk 2 above)
Comment (fsoubelet) [reply docs/packages/omc3/analysis.md:776]: > or with an example, e.g loading them as tfs_df and say that freq_df[10, "BPM123"] has the amplitude amp_df[10, "BPM123"]

I feel like the python example does that.

File: docs/packages/omc3/analysis.md (hunk 3 above)
Comment (fsoubelet) [reply docs/packages/omc3/analysis.md:800]: This is much better, thanks!

File: docs/packages/omc3/analysis.md (hunk 4 above)
Comment (fsoubelet) [reply docs/packages/omc3/analysis.md:801]: Much better too.

File: docs/packages/omc3/analysis.md (hunk 1 above)
Comment (fsoubelet) [reply docs/packages/omc3/analysis.md:794]: I actually had already created an admonition for the spectrum plotter above :)

**Review by JoschD — APPROVED** [review]

**Pull request merged**